corehq/apps/export/models.py

```diff
@@ -135,6 +135,8 @@ def to_json_path(xml_path):
                             meta.options.append(opt['value'])
 
                     self.question_schema[question_path] = meta
+                else:
+                    self.question_schema.pop(question_path, None)
```

Comment:
maybe add a comment about the intention behind this? could be confusing since 99% of the time it does nothing, right?